src/Containers/OhmmsPETE/OhmmsArray.h

```diff
@@ -95,6 +97,74 @@ class Array
     return X.device_data();
   }
 
+  template<typename SIZET = size_t, typename = std::is_integral<SIZET>>
+  Type_t* data(const std::array<SIZET, D>& offsets)
+  {
+    size_t offset = offsets[0];
+    for (int i = 1; i < offsets.size(); i++)
```

Comment:
This is a repetition written differently of the operator() indexing code.  Might be better have it all work the same.  They all calculate a linear index into the data from D indexes.

Reply:
Rewrite `operator()` with parameter packing